client/initializers/underscore.js

```diff
@@ -15,3 +15,69 @@ _.isPresent = function(value) {
 _.isBlank = function(value) {
   return !_.isPresent(value);
 };
+
+// FIXME: Replace this comparator function by server side implementation
+// Based on topic: http://stackoverflow.com/questions/6832596/how-to-compare-software-version-number-using-js-only-number
+_.compareVersion = function(v1, v2) {
```

Comment:
I don't really care, but I would have expected these functions to be defined in a `projectVersions` angular service.